network-api/networkapi/wagtailcustomization/static/wagtailadmin/js/max-length-field.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Even listeners to let editors know when they reached the recommended maximum length of a title.
```

Comment:
```suggestion
 * Event listeners to let editors know when they reached the recommended maximum length of a title.
```

Found a small typo, other than that everything looks great!